services/movie_session.py

```diff
@@ -0,0 +1,47 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: str,
+        cinema_hall_id: int,
+        movie_id: int,
+) -> MovieSession:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
```

Comment:
you can simply pass ids here without creating an instance explicitly: `cinema_hall_id=cinema_hall_id`